templates/zerver/api/changelog.md

```diff
@@ -157,7 +157,8 @@ No changes; feature level used for Zulip 3.0 release.
 * [`POST /register`](/api/register-queue): Added
   `realm_community_topic_editing_limit_seconds` to the response, the
   time limit before community topic editing is forbidden.  A `null`
-  value means no limit.
+  value means no limit. This value is currently not configurable, and hard-coded
+  in the server as 86400 seconds (1 day).
```

Comment:
While it's true that this value is currently not configurable, based on e242ddc8489724d94472b8826ca011de155d05c8 the intent of this setting is to enable it to be changed away from the fixed value that was previously assumed.

Given that this is a changelog, the main information I would like to see added here is what was assumed prior to this change, ie. that fixed 1-day value or `null` (dependent upon the other setting). We could mention that this setting may initially return that value as a default and change in future, but that's not as important from my perspective.